src/parsing/Parse_rule.ml

```diff
@@ -470,14 +470,18 @@ and wrap_type_expr env key lang str =
   match lang with
   (* `x` is a placeholder and won't be used during unwrapping. *)
   | Lang.Java -> spf "(%s x)" str
+  | Lang.Python -> spf "x: %s" str
```

Comment:
maybe we want to move this wrapping/unwrapping in a different file, like Parse_metavariable_type.ml